cmd/githubgen/members.txt

```diff
@@ -0,0 +1,122 @@
+AbhiPrasad
+Aneurysm9
+Caleb-Hurshman
+Doron-Bargo
+Frapschen
+JaredTan95
+MaxKsyunz
+MitchellGale
+MovieStoreGuy
+StefanKurek
+TylerHelmuth
+YANG-DB
+aabmass
+aboguszewski-sumo
+ag-ramachandran
+agoallikmaa
+albertteoh
+alexvanboxel
+altuner
+antonblock
+architjugran
+arminru
+armstrmi
+asaharan
+astencel-sumo
+atingchen
+atoulme
+avadhut123pisal
+bertysentry
+billmeyer
+binaryfissiongames
+bogdandrutu
+boostchicken
+chrroberts-pure
+codeboten
+cpheps
+crobert-1
+damemi
+dao-jun
+dashpole
+dehaansa
+dgoscn
+dineshg13
+djaglowski
+dmitryax
+dyladan
+eedorenko
+emreyalvac
+erichsueh3
+evan-bradley
+fatsheep9146
+frzifus
+gbbr
+gjanco
+gouthamve
+gramidt
+hanjm
+hickeyma
+hughesjj
+hvaghani221
+jacobmarble
+jamesmoessis
+jmacd
+jonathanwamsley
+jpkrohling
+jsuereth
+keep94
+kentquirk
+khyatigandhi6
+kiranmayib
+kkujawa-sumo
+kongluoxing
+leonsp-ai
+liqiangz
+liustanley
+mapno
+mar4uk
+martin-majlis-s1
+matej-g
+mcardy
+mrod1598
+mrsillydog
+mwear
+mx-psi
+mxiamxia
+nslaughter
+oded-dd
+open-telemetry/collector-approvers
```

Comment:
Same as in previous comment, I think we should exclude teams from this tool, since we can easily identify them through the `/`